### ISSUE TITLE
visual indicator of connection strength between client and server

### ISSUE THREAD

**Issue by the-wes** [synergy/synergy#5191]
Problem: Users experience mouse lag or other performance issues for various reasons, but don't have any visibility into what the cause may be.

Solution: We can give them some idea by including an indicator on the client of how long it's taking to get message back and forth from the server. A "ping indicator" or "lag meter" of some kind would work for this.

**Comment by vanowm** [comment]
The problem with lag not necessarily related to network issue. In my case the lag starts after client was running non stop for a few days/weeks and server would go to sleep twice a day. While synergy is laggy to the point that it's unusable, I still can copy files from one computer to another at sustain 110Mb/sec (0.88gbps).
I love synergy, but it's lacking so much common sense...has no indication that it's running (no tray icon, even though there is a non-system process running), saved config files can't be edited via GUI when loaded later (WTF?)

**Comment by yupi2** [comment]
@vanowm Is there any memory leakage (like many GB) for the server or client by any chance?

**Comment by vanowm** [comment]
No, not really, under 15mb (30mb max) on client (w7) and 63mb (74mb max) on server (w10)
Loggin level set to info.
